web/portal/client/cypress/e2e/User/HuntGroups/HuntGroups.cy.js

```diff
@@ -0,0 +1,60 @@
+import UserCollection from '../../../fixtures/Users/getCollection.json';
+
+Cypress.on('uncaught:exception', (error, runnable) => {
+  if (
+    error.message.includes(
+      "Cannot read properties of undefined (reading 'countryCode')"
+    )
+  ) {
+    return true;
+  }
+
+  return true;
+});
```

Comment:
Why is this happening? I think this should be controlled, then remove this command.

Reply:
Upps. I used that for debugging something. It does nothing now: look at line 9. It should return false to ignore the exception.

Deleted